setup/setup.wxs

```diff
@@ -90,6 +85,11 @@
                                 <File Name="OSPSuite.PKParameters.xml" Id="OSPSuite.PKParameters.xml" Source="$(var.DeployDir)/OSPSuite.PKParameters.xml" KeyPath="yes"/>
                             </Component>
 
+
+                            <Component Id='log4net.config.xml' Guid='DFCFEDFA-2B88-4FFD-BA64-D4D108D98FD1'>
```

Comment:
I'd like to test and make sure that this is the right place. Logging is not used in MoBi anyways